transactional_licence_form/forms.py

```diff
@@ -0,0 +1,317 @@
+from django import forms
+
+YES_NO_CHOICES = ["Yes", "No"]
+
+
+def with_label_class(class_name, field):
+    field.label_class = class_name
+    return field
+
+
+def list_to_choices(values):
```

Comment:
I think this may be better suited in a utils file, I've started separating them from a single utils.py file into utils/type_of_utils.py structure so maybe a form_utils file would be the best place for it?

If not, given we are only using the method in this file, could you put a trailing _ before the name to indicate it's private?